src/misc/setup.cpp

```diff
@@ -420,9 +436,10 @@ void Prop_multival::make_default_value()
 }
 
 //TODO checkvalue stuff
-bool Prop_multival_remain::SetValue(std::string const& input) {
-	Value val(input,Value::V_STRING);
-	bool retval = SetVal(val,false,true);
+bool Prop_multival_remain::SetValue(const std::string &input)
```

Comment:
Could fix the casing as it's local only.

Reply:
Ok! Fixed up casing on all the multi-values; rippled a bit, but not a big deal: https://github.com/dosbox-staging/dosbox-staging/pull/1818/commits/eaeb5567b56636ec707c04fb7466b615dda337f5

The property and setup classes can still use more case fixing, but we might as well cut-off the scope of this PR here, and leave renames for a separate PR.

Reply:
Sound like a good idea 👍🏻